packages/form-js-viewer/test/TestHelper.js

```diff
@@ -5,6 +5,9 @@ import { insertCSS } from './helper';
 // @ts-ignore-next-line
 import formCSS from '../dist/assets/form-js.css';
 
+// @ts-ignore-next-line
+import flatpickrCSS from '@bpmn-io/form-js-viewer/dist/assets/flatpickr/light.css';
```

Comment:
Is this only for tests? Was it already included in the actual implementation?

Reply:
This is only for tests, yes. We forgot this one for our test suite, as it does not make a big difference for the `npm run start:viewer` test, only visually. I recognized it while inspecting the viewer test in the browser.

Reply:
Yeah, if this changed anything for the actual implementation we would have to check if it doesn't break the customization on Tasklist